lib/PerfMeter.js

```diff
@@ -0,0 +1,143 @@
+/* globals Services */
+const {Cu} = require("chrome");
+Cu.import("resource://gre/modules/Services.jsm");
+
+const tabs = require("sdk/tabs");
+const simplePrefs = require("sdk/simple-prefs");
+
+const VALID_TELEMETRY_TAGS = new Set([
+  "TAB_READY",
+  "WORKER_ATTACHED",
+  "TOP_FRECENT_SITES_REQUEST",
+  "TOP_FRECENT_SITES_RESPONSE",
+  "RECENT_BOOKMARKS_REQUEST",
+  "RECENT_BOOKMARKS_RESPONSE",
+  "RECENT_LINKS_REQUEST",
+  "RECENT_LINKS_RESPONSE",
+  "FRECENT_LINKS_REQUEST",
+  "FRECENT_LINKS_RESPONSE",
+  "SEARCH_STATE_REQUEST",
+  "SEARCH_STATE_RESPONSE",
+  "NOTIFY_PERFORMANCE",
+]);
+
+function PerfMeter(trackableURLs) {
+  this._trackableURLs = trackableURLs;
+  this.init();
+}
+
+PerfMeter.prototype = {
+
+  _tabs: {},
+
+  get events() {
+    return this._tabs;
+  },
+
+  init: function() {
```

Comment:
FWIW, I believe this is ESLint's [`object-shorthand`](http://eslint.org/docs/rules/object-shorthand) rule.

If I add `"object-shorthand": 1,` to the .eslintrc file, I currently get about 24 warnings [in **master**, I haven't tried against this branch specifically yet].